vmslave-config/ansible/deb_build_vmslave_config.yml

```diff
@@ -1,10 +1,10 @@
 ---
 - hosts: deb
   roles:
-    - replace-apt-source
```

Comment:
Why we replace the EMC internal apt-source ? I remembered you use EMC internal source because external source is very unstable. but why change it back again ?